src/components/menu/menu.css

```diff
@@ -10,6 +10,7 @@
   align-items: center;
   height: 92px;
   box-shadow: 0 2px 88px 0 #fde9da;
+  width: 100%;
```

Comment:
Shouldn't you define the body or html element with "width: 100vw" to assure that the width of the nav-bar will always extend to the screen width? For what I saw you just defined a class (.fullWidth or something) with 100vw.

Reply:
Thank you @RolfBuchwaldAlvarez :)
If we declare `width: 100vw` here, we lose control over the width by a parent element. With this rule (`width: 100%`), we expect the parent element to handle the actual width. 
This way we can control the width at a higher level. `.fullWidth` is used to simulate such a parent element in Storybook. In Material UI for example the `<Container >` or `<Grid>` components would handle this sort of thing.

Reply:
I know. But which parent element here is defined with 100vw? I cannot find any 😅

Reply:
Ah, none.
Since we're using `width: 100%` we can rely on other methods, like `display: grid` in `.grid-container`, whichgives us the full width because there's only one column

Reply:
👍